### PR TITLE
Event Analytics

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/search/get-started-with-search/search-page/log-level.md [hunk 1]
```diff
@@ -60,3 +57,13 @@ The <strong>_loglevel</strong> field value for log messages with the log level <
 The log level of a log line is stored under the `_loglevel` field. You can override its value using a [field extraction rule (FER)](/docs/manage/field-extractions/create-field-extraction-rule/). For example:
 
 <img src={useBaseUrl('img/search/get-started-search/search-page/add-field-extraction-rule.png')} alt="Add field extraction rule" width="400"/>
+
+### Events
+
+Events are a special type of logs that captures any action that can change the state of the system. These events can be used to identify and mitigate the outages in your system. You can view the indication for the events in the histogram, and clicking on this will provide details like **Event start time**, **Event type**, **Event Source**, and **Kubernetes Event Details**.
```
Comment (apoorv-garg) [inline docs/search/get-started-with-search/search-page/log-level.md:63]:
```suggestion
Events are actions that can change the state of the system. These could be code deployments, feature flag changes, infrastructure changes and so on. They can be used to identify and mitigate outages in your system.

Sumo Logic intelligently detects and surfaces related events when you run log searches. You can view the indication for the events in the histogram and the aggregate chart below. By clicking on the event, you can view details like **Event start time**, **Event type**, **Event Source** and other metadata associated with the event etc..
```

File: docs/search/get-started-with-search/search-page/index.md [hunk 2]
```diff
@@ -64,6 +64,12 @@ In this section, we'll introduce the following concepts:
   <p>Learn how to filter results based on a histogram time range.</p>
   </div>
 </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/search/get-started-with-search/search-page/event-analyticsl"><img src={useBaseUrl('img/icons/search.png')} alt="icon" width="40"/><h4>Event Analytics</h4></a>
+  <p>Learn more about events in histogram and aggregate charts for your search results.</p>
+  </div>
+</div>
```
Comment (jpipkin1) [inline docs/search/get-started-with-search/search-page/index.md:72]: Move down after "Field Browser".

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812.

File: docs/search/get-started-with-search/search-page/index.md (hunk 2 above)
Comment (JV0812) [reply docs/search/get-started-with-search/search-page/index.md]: Doc is made as Beta, so we will not update the index page.

**Comment by jpipkin1** [comment]
@JV0812, I blurred out the details on event-analytics.png.

**Comment by JV0812** [comment]
@jpipkin1 Thanks for your review and support in blurring the important details.

File: docs/search/get-started-with-search/search-page/event-analytics.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+id: event-analytics
```
Comment (apoorv-garg) [inline docs/search/get-started-with-search/search-page/event-analytics.md:2]: we should a section describing what event sources are supported. Right now we only support deploy events from k8s events data collected using https://help.sumologic.com/docs/observability/kubernetes/

File: docs/search/get-started-with-search/search-page/event-analytics.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Closed Beta</span></a></p>
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Events are actions that can change the state of the system. These could be code deployments, feature flag changes, infrastructure changes and so on. They can be used to identify and mitigate outages in your system. Currently, we only support to deploy events from the [Kubernetes](/docs/observability/kubernetes/).
```
Comment (apoorv-garg) [inline docs/search/get-started-with-search/search-page/event-analytics.md:16]: nit
```suggestion
Events are actions that can change the state of the system. These could be code deployments, feature flag changes, infrastructure changes and so on. They can be used to identify and mitigate outages in your system. Currently, we only support code deploy events from [Kubernetes](/docs/observability/kubernetes/).
```

File: blog-service/2024-01-19-event-analytics.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: Event Analytics (Search)
```
Comment (apoorv-garg) [inline blog-service/2024-01-19-event-analytics.md:2]: will these release notes go out to all customers ?

Comment (JV0812) [reply blog-service/2024-01-19-event-analytics.md:2]: Yes, this will be published under this https://help.sumologic.com/release-notes-service/ page.

Comment (apoorv-garg) [reply blog-service/2024-01-19-event-analytics.md:2]: i don't think we would want to announce to all customers right now. we can skip the release notes

Comment (JV0812) [reply blog-service/2024-01-19-event-analytics.md:2]: Okay, so you only need the Beta doc right now?

Comment (apoorv-garg) [reply blog-service/2024-01-19-event-analytics.md:2]: correct

Comment (JV0812) [reply blog-service/2024-01-19-event-analytics.md]: Sure, the PR is updated now with just the Beta doc. So when can we publish this PR?

**Pull request merged**